numerics/double_precision.hpp

```diff
@@ -127,6 +135,7 @@ using internal_double_precision::Mod2π;
 using internal_double_precision::TwoDifference;
 using internal_double_precision::TwoProduct;
 using internal_double_precision::TwoSum;
+using internal_double_precision::VeltkampDekkerProduct;
```

Comment:
Why would any client ever want to use this function?  Shouldn't it be purely internal?

Reply:
See the comments:
https://github.com/mockingbirdnest/Principia/blob/19073a6492441fedf411b2107e58edcbc6b105f8/numerics/double_precision.hpp#L55-L57